helm/cloud-provider-vsphere/templates/secret.yaml

```
@@ -1,15 +1,15 @@
-{{- if .Values.config.enabled | default .Values.global.config.enabled -}}
+{{- if and .Values.config.secret.create .Values.config.enabled | default .Values.global.config.enabled -}}
 apiVersion: v1
 kind: Secret
 metadata:
-  name: vsphere-cloud-secret
+  name: {{ .Values.config.secret.name | default "vsphere-cloud-secret" }}
```

```suggestion
  name: {{ .Values.config.secret.name }}
```
It is already set as default value in https://github.com/giantswarm/cloud-provider-vsphere-app/pull/25/files#diff-2c4c70a148aa47d79b8d3a24bae0247c998f22ecfe9e30e8ae59d4ee9c812ac4R23